### PR TITLE
Bug 1734011 - Scrape and expose Glean tags

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: tests/test_git_scraper.py
Comment (wlach) [inline tests/test_git_scraper.py:235]: This comment was wrong

File: tests/resources/test_repo_files/normal/1/metrics.yaml
Comment (wlach) [inline tests/resources/test_repo_files/normal/1/metrics.yaml:12]: Making these changes here so we can still test the same sorts of things.

File: Dockerfile
Comment (wlach) [inline Dockerfile:39]: Obviously this needs to be removed before we merge. Instead we should version bump glean_parser, release a new version, and have probe_scraper depend on it.

File: probeinfo_api.yaml
Comment (badboy) [inline probeinfo_api.yaml:639]: https://github.com/mozilla/glean/pull/1849/files#diff-01a82f719b00c6c1cf94ad9ae35864e3997965fd9bcee99a77a4f5f50c3c3b1bR19-R20

We say they can be named whatever, but then probe-scraper actually needs them with a specific name. We should make sure our docs are consistent on that
(I guess our docs on metrics.yaml/pings.yaml are also wrong then, I'll file a bug)

**Review by badboy — APPROVED** [review]
Preliminary r+.
Blocked by the glean_parser release, but you pointed out all the places that need to be reverted again.

File: probeinfo_api.yaml
Comment (badboy) [reply probeinfo_api.yaml:639]: https://bugzilla.mozilla.org/show_bug.cgi?id=1738880

**Pull request merged**
